msteams-platform/sbs-tabs-and-messaging-extensions-with-sso.yml

```diff
@@ -78,7 +78,7 @@ items:
     1. Enter required bot handle name in **Bot handle**.
     1. Select your **Subscription** from the dropdown list.
     1. From the **Resource group** dropdown list, select your existing resource group. You can also create a new resource group (select **Create new** > enter resource name > select **OK**).
-
+       For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
Link to create resource group is not required here there is a link in the UI to create a resource group. Remove

Reply:
The link is removed as suggested.